$NetBSD: patch-ac,v 1.1 1999/03/05 10:05:46 agc Exp $

Don't try to put dots in front of an absolute pathname.

--- configure	1999/03/05 09:57:49	1.1
+++ configure	1999/03/05 09:59:04
@@ -1996,10 +1996,12 @@
     top_srcdir="$ac_dots$ac_given_srcdir" ;;
   esac
 
+if false; then
   case "$ac_given_INSTALL" in
   [/$]*) INSTALL="$ac_given_INSTALL" ;;
   *) INSTALL="$ac_dots$ac_given_INSTALL" ;;
   esac
+fi
   echo creating "$ac_file"
   rm -f "$ac_file"
   configure_input="Generated automatically from `echo $ac_file_in|sed 's%.*/%%'` by configure."
